sale_delivery_date/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module add the delivery date in sale order and assign delivery date depends on warehouse shipping delay.
```

Comment:
```suggestion
This module does the following:

- Adds the delivery date and time fields in sale.order.
- Updates commitment_date of the order when delivery date is selected, based on the delivery date
  and the warehouse.shipping.delay parameter setting, so that the delivery schedule is adjusted accordingly.
```